tests/digraph/test_bisimulation.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import rustworkx
+
+
+class TestBisimulation(unittest.TestCase):
+    def test_is_maximum_bisimulation(self):
+        for graph, solution in zip(self.graphs, self.reference_solution):
+            res = rustworkx.digraph_maximum_bisimulation(graph)
+            res_format = [tuple(element) for element in res]
+            for element in res_format:
+                self.assertTrue(all([any([el in sol for sol in solution]) for el in element]))
```

Comment:
You can split these tests over multiple functions or at least use `with self.subTest()` to help identify the failures individually